widget/scroller.go

```diff
@@ -444,6 +448,9 @@ func (s *ScrollContainer) updateOffset(deltaX, deltaY int) bool {
 	}
 	s.Offset.X = computeOffset(s.Offset.X, -deltaX, s.Size().Width, s.Content.MinSize().Width)
 	s.Offset.Y = computeOffset(s.Offset.Y, -deltaY, s.Size().Height, s.Content.MinSize().Height)
+	if s.onOffsetChanged != nil {
+		s.onOffsetChanged()
+	}
```

Comment:
```suggestion
	if f := s.onOffsetChanged; f != nil {
		f()
	}
```
It is possible, though unlikely, that the field could be set nil after it is nil checked and before it is called. This suggestion guarantees that cannot happen.

Reply:
This should be fixed